### PR TITLE
Point time series filter populated from map click

### PR DESCRIPTION
**Pull request by pmbohm**
https://github.com/aodn/backlog/issues/418

### CODE REVIEW

**Comment by bhasin85** [comment]
@pmbohm  The co-ordinates do not match when I click within the bounding box. Am I missing something here?

open access to ocean data

**Comment by pmbohm** [comment]
All i good to go now

File: web-app/js/portal/details/PointTimeSeriesPanel.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+                    new Portal.filter.PointFilter({
+                        name: 'timeSeriesAtPoint',
+                        value: pointFilterValue
+                    })
+                );
+            }
+        }
+        // disable 'timeSeriesAtPoint' and re-instate the global map constraint
+        else {
+            var timeSeriesAtPoint = Portal.filter.FilterUtils.getFilter(this.dataCollection.filters,"timeSeriesAtPoint");
```
Comment (bhasin85) [inline web-app/js/portal/details/PointTimeSeriesPanel.js]: Duplicate declaration at line 171

File: web-app/js/portal/details/PointTimeSeriesPanel.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            maxValue : 90,
+            minText : "The minimum allowed latitude (EPSG:4326) is {0}",
+            maxText : "The maximum allowed latitude (EPSG:4326) is {0}",
+            style: 'margin: 0 0 0 15px',
+            listeners: {
+                scope: this,
+                render: function(c) {
+                    this._addTimeSeriesQuickTip(c);
+                },
+                change: function(c) {
```
Comment (bhasin85) [inline web-app/js/portal/details/PointTimeSeriesPanel.js]: Unused parameter c

File: web-app/js/portal/details/PointTimeSeriesPanel.js [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+            minValue : -180,
+            maxValue : 180,
+            minText : "The minimum allowed longitude (EPSG:4326) is {0}",
+            maxText : "The maximum allowed longitude (EPSG:4326) is {0} \n(Limit is 180)",
+            listeners: {
+                scope: this,
+                render: function(c) {
+                    this._addTimeSeriesQuickTip(c);
+                },
+                change: function(c) {
```
Comment (bhasin85) [inline web-app/js/portal/details/PointTimeSeriesPanel.js]: Unused parameter c

File: web-app/css/general.css [hunk 4]
```diff
@@ -1300,7 +1296,25 @@ span.menu-title {
...
+
+input:read-only { background: #eee; }
+
+.x-trigger-noedit {
+    cursor: default;
+}
+
+.spatialSubsetControl {
+    padding: 15px 5px 15px 10px;
+}
```
Comment (bhasin85) [inline web-app/css/general.css]: Maybe add a newline

**Comment by bhasin85** [comment]
@pmbohm  Finished review. It works flawlessly. :+1:

**Comment by pmbohm** [comment]
@bhasin85 Thanks

**Pull request merged**